openreview/venue_request/venue_request.py

```diff
@@ -714,10 +714,19 @@ def setup_decision_stage(self):
                 'order': 15,
                 'required': True
             },
-            'decision_options': {
```

Comment:
I would prefer to keep decision options and have a new field to enter the ones that are considered acceptance. This way you don't need to make so many changes. 

When you evaluate if a decision is acceptance you just check if the decision is in the accept_decision_options instead of checking the string has an "Accept" token.

Reply:
I think if we completely remove the "Accept" string checking logic then API 1 tests will fail, what do you think?

Reply:
yes, you should support both.

If accept_decision_options is present then check the decision is in the array.
if accept_decision_options is not present then check "Accept" is in the decision string.

Reply:
I pushed some updated changes from your feedback. I forgot to add the preprocess validation for the decisions, I'll add that next.

Reply:
Changes look good! it seems the build is failing, I'm not sure if it is related to these changes....

Reply:
Oops, I set the wrong default value in `invitation_edit_process.py`, should be fixed now.